src/celeritas/optical/model/RayleighModel.cc

```diff
@@ -111,6 +105,27 @@ void RayleighModel::build_mfps(OptMatId mat, MfpBuilder& build) const
         }
         build(grid);
     }
+    // Build a grid with infinite MFP to prevent selection of the model
+    else
+    {
```

Comment:
Comments belong inside the block to which they apply
```suggestion
    else
    {
        // Build a grid with infinite MFP to prevent selection of the model
```